app/assets/stylesheets/application.css.scss

```diff
@@ -5,7 +5,6 @@
  *= require_tree .
```

Comment:
I erred on the side of hacking things into the css here, on the assumption that we would eventually be reorganizing this and moving it to webpacker, but I AM leaving a mess here. So we could also clean it up a little bit as part of this process, start sticking global styles in their own sheet, etc. But it's also not strictly necessary for BS4, so I will leave this call up to the reviewer - hacky CSS okay, or should we start cleaning things up while we're wrenching around in here?

Reply:
This is a big enough chunk of work already that I'm pretty on board with Doing It Later, so long as we aren't making more work for us in the process.

Reply:
popped this into #1854